texar/modules/__init__.py

```diff
@@ -15,6 +15,7 @@
 Modules of Texar library module.
 """
 
+from texar.modules.berts import *
```

Comment:
Can we use some other name to replace `berts`? This sounds really strange...

I suggest calling this package along the lines of `representations` or `pretrained`. If we plan to add ELMo, XLNet and friends, they could go here as well.

Reply:
yes, `pretrained` looks like a good name. @gpengzhi can you update Texar-TF as well